### PR TITLE
Added more authors and some details.

### PR DESCRIPTION
**Pull request by thecsw**
The readme is a bit outdated.

### CODE REVIEW

**Comment by TwiN** [comment]
> Our math guy. Can make really pretty graphs

@jimbobur

LOL

**Comment by jimbobur** [comment]
Are we allowed youtube links in the authors list? If so: https://youtu.be/sIlNIVXpIns

**Comment by robalb** [comment]
maybe you should say 
  WARNING! The system will be rebooted soon.
instead of 'rebooted now '
and avoid repeating it again at the end of the phrase

**Comment by jimbobur** [comment]
And also maybe dial back the claims about how great the new formula is gonna be. I'm not saying it won't, but if it isn't or there's some problem/revision to be made..

**Comment by robalb** [comment]
Check this https://github.com/MemeInvestor/memeinvestor_bot/issues/82#issuecomment-395108523

**Comment by thecsw** [comment]
@jimbobur I actually thought of this https://youtu.be/xbiDrzTd8fE, the math guy

File: src/message.py [hunk 1]
```diff
@@ -10,6 +10,9 @@
 
 help_mess = """
 ^For ^more ^information ^see ^[memes.market](https://memes.market/#info1)
+_______________________
+
+WARNING! The system will be rebooted now. The whole market is bloated and biased. The formula will be changed. All accounts will be kept but their balances will be reset to 1000 MemeCoins. New formula can guarantee natural and fair investments, without billions and trillions in a couple of days. Investors with more than 1 billion will receive some cosmetic rewards to compensate the loss. (| || || |_) The reboot will happen soon. For more information please see the website.
```
Comment (rickles42) [inline src/message.py:15]: This seems like quite a long message to include in every response by the bot. I think if you plan to make a sticky post announcing the upcoming reset, then this message in the response text could be removed, or just replaced with a very brief pointer to the post.

File: README.md [hunk 2]
```diff
@@ -93,54 +95,19 @@ how to deploy the project on a live system.
 
 ### Prerequisites
 
-In order to run this application, you need to install [praw](https://github.com/praw-dev/praw), Python Reddit API Wrapper. This will be the main and only package to connect to Reddit's API and extract desired data.
```
Comment (rickles42) [inline README.md]: Apologies but I've rewritten these setup instructions based on my work to make it easier for newcomers to launch the bot with Docker. Hence the merge conflict here.

File: src/message.py (hunk 1 above)
Comment (thecsw) [reply src/message.py:15]: Yeah. For now, I think we should just warn in our comments. I will make a sticky later!

**Comment by thecsw** [comment]
Conflicts resolved, merging this baby.

**Pull request merged**